config/config.go

```diff
@@ -107,6 +107,7 @@ type (
 var DefaultWorkflowConfig = &iwfidl.WorkflowConfig{
 	ContinueAsNewThreshold:       iwfidl.PtrInt32(100),
 	DisableSystemSearchAttribute: iwfidl.PtrBool(true),
+	ExecutingStateIdMode:         iwfidl.PtrString("ENABLED_FOR_STATES_WITH_WAIT_UNTIL"),
 }
```

Comment:
I think we can just remove both and only keep `ContinueAsNewThreshold`. 
`ENABLED_FOR_STATES_WITH_WAIT_UNTIL` is the default behavior already, we don't have to pass it. It will be passed through workflow input, which will consume a some temporal storage 🤦 , for `ExecutingStateIdMode: "ENABLED_FOR_STATES_WITH_WAIT_UNTIL",` it will be 60 bytes. and if a workflow execution run for a month, it will be 60 bytes * month Lol. 

The reason that I added `DisableSystemSearchAttribute` is that the default behavior was different, and I didn't have time to change the default behavior using versioning.

Reply:
Done